multiuser/sql-schema/src/main/resources/che-schema/5.19.0/0.1__add_permissions.sql

```diff
@@ -1,5 +1,5 @@
 --
--- Copyright (c) 2012-2018 Red Hat, Inc.
+-- Copyright (c) 2012-2017 Red Hat, Inc.
```

Comment:
it seems that there are some leftover 2017 dates on these sql files that should not have been changed

Reply:
@benoitf you're right. These changes must be excluded from this PR since `flyway` evaluates hash of content and uses it to ensure that the right migration scripts are applied.

Reply:
in fact it's ok I read too fast :-) Thomas is putting back the dates to avoid master changes. please ignore my remark

Reply:
=) As well as my comment ) I saw that these sql files are modified and missed that this PR is not targetted to master branch =)
So, please mark this thread as resolved )